README.md

```diff
@@ -26,13 +26,15 @@ The examples are configured using environment variables. You can set these manua
 ```bash
 $ export CLOUDSMITH_API_KEY=YOUR_API_KEY_HERE
 $ export CLOUDSMITH_REPOSITORY=your-name-or-org/your-repository
+$ export CLOUDSMITH_REPOSITORY_IS_PRIVATE=yes/no
```

Comment:
Are the actual correct values "yes" or "no", or is it something different like YES/NO or true/false?

I don't currently know the answer to that.